tests/src/test_projects/storagemapvec/src/main.sw

```diff
@@ -0,0 +1,67 @@
+contract;
+
+use sway_libs::storagemapvec::StorageMapVec;
+
+storage {
+    mapvec: StorageMapVec<u64, u64> = StorageMapVec {},
+}
+
+abi TestContract {
+    #[storage(read, write)]
+    fn push(key: u64, value: u64);
+
+    #[storage(read, write)]
+    fn pop(key: u64) -> Option<u64>;
+
+    #[storage(read)]
+    fn get(key: u64, index: u64) -> Option<u64>;
+
+    #[storage(read)]
+    fn len(key: u64) -> u64;
+
+    #[storage(read)]
+    fn is_empty(key: u64) -> bool;
+
+    #[storage(write)]
+    fn clear(key: u64);
+
+    // #[storage(read)]
```

Comment:
Is there a reason to why this is commented out?

Reply:
Cannot test the to_vec function as SDK support does not exist for returning Vecs

Reply:
If this can't be tested, then I'm not sure the function should then be added to the library. The function is unable to return Vecs to the SDK and [returning to an external abi](https://github.com/FuelLabs/sway/issues/3397) call is also not supported. Can you comment it out and provide a github link to the SDK issue so it's easily trackable?

Reply:
As a workaround, can you test this by returning a short Vec element by element as a tuple ?